1.train_models/utils/PlateTrainingSplits.py

```diff
@@ -0,0 +1,62 @@
+class PlateTrainingSplits:
```

Comment:
Not sure if you wanted to add a docstring for this class?? Up to you, the class method doc strings look good

Reply:
Added